packages/ui/pod-pcd-ui/package.json

```diff
@@ -21,6 +21,7 @@
   "scripts": {
     "lint": "eslint \"**/*.ts{,x}\"",
     "build": "tsc -b tsconfig.cjs.json tsconfig.esm.json",
+    "watch": "tsc --watch -p tsconfig.cjs.json & tsc --watch -p tsconfig.esm.json",
```

Comment:
prefer to use the package `concurrently`, otherwise this background process has a chance of existing indefinitely